### PR TITLE
Remove remaining references to galaxy.eggs . flake8 some tools.

### PR DESCRIPTION
**Pull request by nsoranzo**
Also update `doc/source/lib` .

### CODE REVIEW

**Comment by martenson** [comment]
I like how you protect code balance @nsoranzo :)
`+1,044 −1,044`

**Comment by nsoranzo** [comment]
@martenson :laughing: I haven't noticed that!

File: lib/pulsar/client/transport/curl.py [hunk 1]
```diff
@@ -1,11 +1,5 @@
 import logging
 
-try:
-    from galaxy import eggs
-    eggs.require("six")
```
Comment (natefoo) [inline lib/pulsar/client/transport/curl.py:5]: This probably ought to remain until galaxyproject/pulsar#85 is finished, and then the Pulsar client lib should be updated from Pulsar once it's done.

Comment (nsoranzo) [inline lib/pulsar/client/transport/curl.py:5]: @natefoo Actually this is a leftover from your commit https://github.com/galaxyproject/galaxy/commit/033b1e88da99b86ffbd33321374aa59d1d72484c#diff-8ce0f1f93264f339e0bf364c7c19d3bd :laughing:

Comment (nsoranzo) [inline lib/pulsar/client/transport/curl.py:5]: galaxyproject/pulsar#85 has been merged.

Comment (natefoo) [inline lib/pulsar/client/transport/curl.py:5]: Huh, I thought I reverted that to the Pulsar version.

**Pull request merged**